### DISCUSSION TITLE
Welcome to LibreTube Discussions!

### DISCUSSION THREAD

**Discussion by Bnyro** [libre-tube/LibreTube#462] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Silther** [comment]
I would be interested to know where feature requests are welcome? There seems to be both the feature request template in the issues and the ideas label in the discussions, or what is the difference?

**Reply by Bnyro** [reply]
Feature requests should be placed under issues if they are likely to be useful for all users.
For changes to the current behavior, a discussion should be opened first.
